core-charter.txt

```diff
@@ -1,40 +1,65 @@
+As part of the framework for building these applications, the CoRE WG has defined the Constrained Application Protocol (CoAP) for the manipulation of Resources on a Device. CoAP is designed for use between Devices on the same constrained network, between Devices and general nodes on the Internet, and between Devices on different constrained networks both joined by an internet.
 
 There also may be intermediary nodes acting as proxies that possibly also interconnect between other Internet protocols and the Devices using the CoAP protocol. It is worth noting that a proxy does not have to occur at the boundary between the constrained network and the more general network, but can be deployed at various locations in the less-constrained network. The CoRE WG will continue to evolve the support of proxies for CoAP to increase their practical applicability.
 
 CoAP supports various forms of "caching". Beyond the benefits of caching already well known from REST, caching can be used to increase energy savings of low-power nodes by allowing them to be normally-off [RFC 7228]. For example, a temperature sensor might wake up every five minutes and send the current temperature to a proxy that has expressed interest in notifications. When the proxy receives a request over CoAP or HTTP for that temperature resource, it can respond with the last notified value, instead of trying to query the Device which may not be reachable at this time. The CoRE WG will continue to evolve this model to increase its practical applicability.
 
-The CoRE WG will perform maintenance as well as possible updating and complementing on its first four standards-track specifications as required:
+The CoRE WG has completed a set of standards-track specifications building on the CoAP specification [RFC 7252] and will perform maintenance as well as possible updating and complementing the framework in areas of resource discovery, group communication, data formats, operation and management, and CoAP related security.
 
-- RFC 6690
-- RFC 7252
-- RFC 7641
-- RFC 7959
+Candidate work items include:
 
-The CoRE WG will continue to evolve the support for CoAP group communication originally developed in the Experimental RFC 7390. The CoRE WG will not develop a solution for reliable multicast delivery of CoAP messages, which will remain Non-Confirmable when sent over multicast.
+* evolve the support for CoAP group communication originally developed in the Experimental RFC 7390(but not develop a solution for reliable multicast delivery of CoAP messages)
```

Comment:
I think that this work should have been parcelled out to another WG (perhaps ACE, perhaps a new WG).
I don't think that we should keep this work in CORE, going forward.

Reply:
Why?  Group communication is one of the unique features of CoAP, and it should be further developed with the rest of CoAP.
(ACE is completely unrelated to this subject.)